docs/esa/hubble/hubble.rst

```diff
@@ -430,11 +359,115 @@ and a set of criteria to filter the results.
 This will perform a cone search with radius 7 arcmins around the target (defined by
 its coordinates or its name) using the filters defined when executing the function.
 
-This function allows the same parameters than the search criteria (see Section 5).
+This function allows the same parameters than the search criteria (see Section 2).
+
+--------------------------
+5. Getting Hubble products
+--------------------------
+
+.. warning::
+
+   Please bear in mind that the default format to download
+   sets of files has been modified from TAR to ZIP.
 
+After retrieving the metadata, the user can filter the result table and get the rows of interest.
+The most important column is 'observation_id' and it is possible to use it to retrieve all the
+associated files.
+
+In eHST is it possible to download products based on their observation ID (mandatory) and
+a required calibration_level (RAW, CALIBRATED, PRODUCT or AUXILIARY) and/or product type (SCIENCE, PREVIEW, THUMBNAIL or AUXILIARY).
+
+Deprecation Warning: product types PRODUCT, SCIENCE_PRODUCT or POSTCARD are no longer supported. Please modify your scripts accordingly.
+
+For instance, next commands will download all files for the raw calibration level
+of the observation 'j6fl25s4q' and it will store them in a file called
+'raw_data_for_j6fl25s4q.zip'.
+
+.. doctest-remote-data::
+
+  >>> from astroquery.esa.hubble import ESAHubble
+  >>> esahubble = ESAHubble()
+  >>> esahubble.download_product(observation_id="j6fl25s4q", calibration_level="RAW",
+  ...                            filename="raw_data_for_j6fl25s4q")  # doctest: +IGNORE_OUTPUT
```

Comment:
Although we're ignoring the outputs (but only do that if frequent changes are expected, e.g. more observations are coming in, the order of the result is not deterministic, etc), please include a few line long of the expected outcome. E.g. in this case, as I see it will be only the string `raw_data_for_j6fl25s4q.zip`

OTOH, all these download examples are actually downloading the files, so I wonder whether cutting back on the actual code examples would make sense (after all your narrative text does the explanation quite well)

Reply:
I added the expected output. Thanks!